frontend/packages/local-storage-operator-plugin/src/components/local-volume-set/local-volume-set-inner.tsx

```diff
@@ -212,14 +208,20 @@ export const LocalVolumeSetInner: React.FC<LocalVolumeSetInnerProps> = ({
               fieldId="create-lvs-max-disk-size"
               className="lso-create-lvs__disk-size-form-group-max-min-input"
             >
-              <TextInput
-                type={TextInputTypes.text}
-                id="create-lvs-max-disk-size"
-                value={state.maxDiskSize}
-                validated={state.isValidMaxSize ? 'default' : 'error'}
-                className="lso-create-lvs__disk-size-form-group-max-input"
-                onChange={onMaxSizeChange}
-              />
+              <Tooltip
```

Comment:
can we make these validations generic so it can be used with ant text input?

Reply:
we can take it up as a refactor in the future where we write a generic component for this. For now, let's just fix the bug.